streams/src/main/java/org/apache/kafka/streams/state/internals/ThreadCache.java

```diff
@@ -43,7 +44,7 @@
     // internal stats
     private long numPuts = 0;
     private long numGets = 0;
-    private long numEvicts = 0;
+    private AtomicLong numEvicts = new AtomicLong(0);
```

Comment:
why make this atomic, we're still only ever evicting/accessing this from the actual StreamThread right?